hedera-node/src/test/java/com/hedera/services/tokens/HederaTokenStoreTest.java

```diff
@@ -369,20 +377,21 @@ public void wipingRejectsMissingAccount() {
 	@Test
 	public void wipingRejectsTokenWithNoWipeKey() {
 		// when:
-		var status = subject.wipe(sponsor, misc, false);
+		var status = subject.wipe(sponsor, misc, 0, false);
 
 		// expect:
 		assertEquals(TOKEN_HAS_NO_WIPE_KEY, status);
 		verify(account, never()).wipeTokenRelationship(misc);
 	}
 
 	@Test
+	@Disabled
```

Comment:
This test should not be disabled. 
Wipe operation on the treasury account should not be allowed.